### ISSUE TITLE
Installation for all users

### ISSUE THREAD

**Issue by ThomasTubato** [UnamSanctam/UnamBinder#34]
Is it possible to add the checkbox with the installation option for all users in the UnamBinder project?

**Comment by UnamSanctam** [comment]
There isn't an installation option in the binder currently, it's not really a common thing to have in a binder since the bound files usually have their own installation (like the miner), what do you need the installation for?

**Comment by ThomasTubato** [comment]
So is there a way to run your own executable for all users?
for example by using this (https://github.com/M2TeamArchived/NSudo) running it as 'System' is there a chance that it will run for all users?

**Comment by UnamSanctam** [comment]
You mean installing something for all users? Then yes adding it into Task Scheduler as 'System' would do that, but you can't really "run" something on all users in Windows.

**Comment by ThomasTubato** [comment]
Yes I know you can't run it on all users however I know there is a possibility (as in your miner) to install the executable for all users. So I wanted to know if by starting the executable as SYSTEM this would install on all users or not.

**Comment by UnamSanctam** [comment]
Possibly, depends on how the creators of the program you're binding adds it into the Task Scheduler, but starting something on the System user (without adding it into the Task Scheduler and then starting that) can be quite difficult.

**Comment by ThomasTubato** [comment]
So if you add the executable on Task Scheduler it starts for all users at startup?

**Comment by UnamSanctam** [comment]
If you specify it to start under the System user when creating the task then yes.

**Comment by ThomasTubato** [comment]
Okey, thanks
